app/controllers/tasks_controller.rb

```diff
@@ -60,11 +60,11 @@ def create_task
   end
 
   def update_task
-    ::Tasks::Update.call(task_params: task_params, task: @task)
+    ::Tasks::Update.call(task_params: task_params, task: @task, project: @project)
```

Comment:
In interactor you can takes project from task, `task.project`
```suggestion
    ::Tasks::Update.call(task_params: task_params, task: @task)
```